package/rpm/install_section

```diff
@@ -3,9 +3,15 @@ python3 setup.py install --single-version-externally-managed -O1 --root=%{buildr
 install -m 0644 -D package/systemd/decisionengine.service %{buildroot}/usr/lib/systemd/system/decisionengine.service
 echo "%attr(0644,root,root) /usr/lib/systemd/system/decisionengine.service" >> INSTALLED_FILES
 
+install -m 0644 -D package/systemd/decisionengine_sysuser.conf %{buildroot}/usr/lib/sysusers.d/decisionengine.conf
+echo "%attr(0644,root,root) /usr/lib/sysusers.d/decisionengine.conf" >> INSTALLED_FILES
+
 mkdir -p %{buildroot}/%{_localstatedir}/log/decisionengine
 echo "%attr(0750,decisionengine,decisionengine) /%{_localstatedir}/log/decisionengine" >> INSTALLED_FILES
 
+mkdir -p %{buildroot}/%{_sharedstatedir}/decisionengine
+echo "%attr(0750,decisionengine,decisionengine) /%{_sharedstatedir}/decisionengine" >> INSTALLED_FILES
```

Comment:
%{_sharedstatedir} (same as %{_localstatedir} in the lines above) is including also the slash at the beginning. There will be no error since // -> /, but it is not needed. Is it added for clarity in the reading?

Reply:
Mostly this was me using the wrong `cw` value in vim, fixed.